script/Arcueid/Day 13/13_00_ARC13_KRAKE.txt

```diff
@@ -112,7 +112,7 @@ Sparks crackle across the concrete face of the building.
 [sha:7f2b4a0503846bbea0b1ab8d39890e9dc687ab4a]{
 -- Page 266, Offset 32397.
 -- ロアがここまで貯蔵した<魔|オ><力|ド>と、
-The magical <energy|o d   > that Roa has been gathering up until now―――
+The <magical energy|o d   > that Roa has been gathering up until now―――
```

Comment:
i think that's intended

Reply:
it would read "Magical Od" which isn't correct

Reply:
If I remember correctly this was something Ben wanted changed simply because it looks so incredibly bad when displayed on screen. Od is so short that it fits in the space between 'magical' and 'energy' and it looks kinda shit.

Could try and remake this sentence so you don't need the Ruby at all, that would be the most preferrable.